docs/src/index.md

```diff
@@ -10,11 +14,20 @@ Pkg.add("StructuralIdentifiability")
 ```
 
 ## Citation
-
+```latex
+@article{structidjl,
+  author  = {Dong, R. and Goodbrake, C. and Harrington, H. and Pogudin G.},
+  title   = {Structural identifiability via input-output projections},
+  journal = {Manuscript in preparation},
+  year    = {2021}
+}
+```
 ## Feature Summary
-`StructuralIdentifiability.jl` can assess local and global identifiability of ODE models. 
+`StructuralIdentifiability.jl` can assess local and global identifiability of ODE models. In addition to these straightforward identifiability queries on individual parameters, the package is able to distinguish between single- and multi-experiment identifiability.
 ## Feature List
-
+* Local identifiability checks (single- and multi-experiment)
+* Global identifiability checks (single- and multi-experiment)
```

Comment:
I would be more careful here as for global identifiability we have
 - ME 
 - and a check ME = SE
 For most of the practical purposes this seems to be fine.
I would just remove the text in the parenthesis